tests/_support/Config/Services.php

```diff
@@ -43,4 +44,20 @@ public static function uri(?string $uri = null, bool $getShared = true)
 
         return new URI($uri);
     }
+
+    /**
+     * The URI class provides a way to model and manipulate URIs.
+     *
+     * @param mixed $getShared
+     *
+     * @return stdClass
+     */
+    public static function someService($getShared = true)
+    {
+        if ($getShared) {
+            return static::getSharedInstance('someService');
+        }
+
+        return new stdClass();
+    }
```

Comment:
By the way, why do you need to add a service?
We can use the existing service for testing.